pjnath/src/pjnath-test/stun_sock_test.c

```diff
@@ -25,8 +25,7 @@ enum {
     RESPOND_STUN    = 1,
     WITH_MAPPED	    = 2,
     WITH_XOR_MAPPED = 4,
-
-    ECHO	    = 8
+    SEND_ECHO	    = 8
```

Comment:
Looks like name clash prevention, but apparently unrelated of mingw64? It compiles fine both midified and unmodified. The only place I see ECHO defined is netmon.h of i686-w64-mingw32, but apparently netmon.h is not used in pjsip? Maybe this change was intended for some other PR?

Reply:
Got compile error previously (yes, name clash), but it was perhaps because of using wrong headers/toolchains (was using msys's instead of mingw's at some point). Anyway, just tested again without the change and it compiles fine now indeed, so will revert the change. Thanks.